src/sync/models/vote.js

```diff
@@ -29,6 +29,7 @@ class Vote {
       txid: this.txid,
       voterAddress: this.participant,
       voterQAddress: Decoder.toQtumAddress(this.participant),
+      topicAddress: '',
```

Comment:
this line can be removed

Reply:
We actually need this since I put `topicAddress` as a required String. Actually now that I think about it, this will break existing users that have vote data already. I think we need to put `topicAddress` as `optional`.